content/docs/guides/pipelines/deploy-pipelines-service.md

```diff
@@ -3,195 +3,10 @@ title = "Set Up and Deploy the Pipelines Service"
-Kubeflow Pipelines along with all dependencies in the cluster. Wait for the 
-number of successful job runs to reach 1:
-
-```
-NAME                      DESIRED   SUCCESSFUL   AGE
-deploy-ml-pipeline-wjqwt  1         1            9m
-```
-
-You can check the deployment log for any failures:
-
-```bash
-kubectl logs $(kubectl get pods -l job-name=[JOB_NAME] -o jsonpath='{.items[0].metadata.name}')
-```
-
-By default, the Kubeflow Pipelines service is deployed with usage collection 
-turned on. 
-It uses [Spartakus](https://github.com/kubernetes-incubator/spartakus) which 
-does not report any personal identifiable information (PII).
-
-When deployment is successful, forward a local port to visit the Kubeflow 
-Pipelines UI dashboard:
-
-```bash
-export NAMESPACE=kubeflow
-kubectl port-forward -n ${NAMESPACE} $(kubectl get pods -n ${NAMESPACE} --selector=service=ambassador -o jsonpath='{.items[0].metadata.name}') 8080:80
-```
-
-Open your browser and point to 
-[localhost:8080/pipeline](http://localhost:8080/pipeline).
-
-## Run your first pipeline
-
-Click through to the **Pipelines** section in the UI, create a new experiment 
-and run a sample pipeline. 
-For the project name parameter, use your GCP project name. 
-
-To build your own pipelines, see the 
-[guide to building a pipeline](/docs/guides/pipelines/build-pipeline).
-
-## Disable usage reporting
-
-If you want to turn off the usage report, you can download the bootstrapper 
-file and change the arguments to the deployment job.
-
-For example, download bootstrapper:
-
-```bash
-PIPELINE_VERSION=0.0.42
-curl https://storage.googleapis.com/ml-pipeline/release/$PIPELINE_VERSION/bootstrapper.yaml --output bootstrapper.yaml
-```
-
-Update the argument in the file:
-
-```
-        args: [
-          ... 
-          # uncomment following line
-          "--report_usage", "false",
-          ...
-        ]
-```
-
-Create a job using the updated YAML by running the following command:
-
-```bash
-kubectl create -f bootstrapper.yaml
-```
-
-## Uninstall
-
-To uninstall Kubeflow Pipelines, download the bootstrapper file and change the 
-arguments to the deployment job.
-
-For example, download bootstrapper:
-
-```bash
-PIPELINE_VERSION=0.0.42
-curl https://storage.googleapis.com/ml-pipeline/release/$PIPELINE_VERSION/bootstrapper.yaml --output bootstrapper.yaml
-```
-
-Update the argument in the file:
-
-```
-        args: [
-          ... 
-          # uncomment following line
-          "--uninstall",
-          ...
-        ]
-```
-
-Create a job using the updated YAML by running the following command:
-
-```bash
-kubectl create -f bootstrapper.yaml
-```
+Starting from [V0.1.3](https://github.com/kubeflow/pipelines/releases/tag/0.1.3), Kubeflow Pipeline becomes one of the Kubeflow core components. It's automatically deployed during Kubeflow deployment. Please refer to [GKE setup](/docs/started/getting-started-gke/)" for setting up Kubeflow.
```

Comment:
s/Kubeflow Pipeline/Kubeflow Pipelines

Reply:
Also remove the extra quotation mark " after the bracket - it's rendering on the page.

Reply:
done. thanks!